src/http/common.cr

```diff
@@ -429,11 +432,10 @@ module HTTP
   end
 end
 
-{% unless flag?(:win32) %}
-  require "./status"
-  require "./request"
-  require "./client/response"
-  require "./headers"
-  require "./content"
-  require "./cookie"
-{% end %}
+require "./status"
+require "./request"
+require "./client/response"
+require "./headers"
+require "./content"
+require "./cookie"
+require "./formdata"
```

Comment:
How about `require "./**"` to make sure everything's required?

Reply:
That would break on windows.

Reply:
why??

Reply:
I'd like to avoid wildcard requires because it can lead to weird issues based on include-order. Writing a few explicit requires isn't bad IMO.